e2e/e2e.js

```diff
@@ -263,179 +265,406 @@ describe('create, edit, get', () => {
+      expect(res.ok).toBe(true)
+      expect(res.status).toBe(200)
+      expect(res.statusText).toBe('OK')
+      expect(Array.isArray(res.body)).toBe(true)
+      expect(res.body[0].id_workspace).toEqual(workspaceId)
+      expect(res.body[0].id_integration).toEqual(credentialId)
+      expect(res.body[0].flow_type).toEqual('adobeid')
+      expect(res.body[0].integration_type).toEqual('oauthweb')
+    })
+
+    test('test subscribeCredentialToServices API (Adobe Stock)', async () => {
+      expect(credentialId).toBeDefined() // if not, createAdobeIdCredential test failed
+      expect(orgId).toBeDefined()
+      expect(projectId).toBeDefined()
+      expect(workspaceId).toBeDefined()
+
+      const sdkCode = findSDKCode('Adobe Stock')
+      const res = await sdkClient.subscribeCredentialToServices(orgId, projectId, workspaceId, 'adobeid', credentialId, [
+        {
+          sdkCode,
+          licenseConfigs: null,
+          roles: null
+        }
+      ])
+      expect(res.ok).toBe(true)
+      expect(res.status).toBe(200)
+      expect(res.statusText).toBe('OK')
+      expect(typeof (res.body)).toBe('object')
+      expect(res.body).toEqual({ sdkList: [sdkCode] })
+    })
+
+    test('test getWorkspaceForCredential API', async () => {
+      expect(credentialId).toBeDefined() // if not, createAdobeIdCredential test failed
+      expect(orgId).toBeDefined()
+      expect(projectId).toBeDefined()
+      expect(workspaceId).toBeDefined()
+
+      const res = await sdkClient.getWorkspaceForCredential(orgId, credentialId)
+      expect(res.ok).toBe(true)
+      expect(res.status).toBe(200)
+      expect(res.statusText).toBe('OK')
+      expect(typeof (res.body)).toBe('object')
+      expect(res.body.projectId).toEqual(projectId)
+      expect(res.body.workspaceId).toEqual(workspaceId)
+    })
+
+    // organization integration APIs on workspace credentials
+    test('test getIntegration API', async () => {
+      expect(credentialId).toBeDefined() // if not, createEnterpriseIntegration test failed
+      expect(orgId).toBeDefined()
+      const res = await sdkClient.getIntegration(orgId, credentialId)
+      expect(res.ok).toBe(true)
+      expect(res.status).toBe(200)
+      expect(res.body.id).toEqual(credentialId)
+      expect(res.body.orgId).toEqual(orgId)
+      expect(res.body.name).toEqual(credentialNameAdobeId)
+      expect(res.body.type).toEqual('adobeid')
+    })
+
+    test('test getIntegrationSecrets API', async () => {
+      expect(credentialId).toBeDefined() // if not, createEnterpriseIntegration test failed
+      expect(orgId).toBeDefined()
+      const res = await sdkClient.getIntegrationSecrets(orgId, credentialId)
+      expect(res.ok).toBe(true)
+      expect(res.status).toBe(200)
+      expect(typeof res.body).toBe('object')
+      expect(res.body.client_id).toBeDefined()
+      expect(res.body.client_secrets).toBeDefined()
+    })
+
+    // atlas policy apis - commented out because return 405 method not allowed ?
```

Comment:
is this one of the bugs you filed? (maybe have a reference to internal bug number)

Reply:
yup IOC-4290